### DISCUSSION TITLE
Using homebrew 🍺  to manage dependencies in git repo, share your experience/thoughts?

### DISCUSSION THREAD

**Discussion by OliverKoo** [Homebrew/discussions#1124] [Everyday usage]
Thinking about using homebrew to manage dependencies in our git repo that is geared toward iOS app development. We have around 30-50 developers working on this repository.  My thoughts is to create our custom tap for formulae that our team wish to control versions of (describe by the [official doc](https://docs.brew.sh/Versions)) 🍺 

Love to hear your experience/challenges/suggestions 😄  or links to guide and resources you found useful.

**Comment by Bo98** [comment]
When you have all the formulae you need, you may find [`brew bundle`](https://github.com/Homebrew/homebrew-bundle) useful as a nice one-command way of installing them all.
